packages/shorebird_cli/test/src/commands/account/create_account_command_test.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:mason_logger/mason_logger.dart';
+import 'package:mocktail/mocktail.dart';
+import 'package:shorebird_cli/src/auth/auth.dart';
+import 'package:shorebird_cli/src/commands/account/account.dart';
+import 'package:shorebird_code_push_client/shorebird_code_push_client.dart';
+import 'package:test/test.dart';
+
+class _MockAuth extends Mock implements Auth {}
+
+class _MockLogger extends Mock implements Logger {}
+
+class _MockUser extends Mock implements User {}
+
+void main() {
+  const userName = 'John Doe';
+  const email = 'tester@shorebird.dev';
+
+  late Auth auth;
```

Comment:
minor but I think in other tests we keep these scoped to the group they correspond to we should just be consistent if possible 🤷